api/services/freshtrakService.ts

```diff
@@ -39,7 +36,7 @@ async function getFTLocationData(site_id: string, zip: string): Promise<freshtra
   
   if(freshTrakAgencyID) {
     agencyURL = FRESHTRAK_AGENCY_URL + freshTrakAgencyID;
-    const response: freshtrakResponseDto = await getFreshTrakEvents(freshTrakAgencyID);
+    const response: freshtrakResponseDto | null = await getFreshTrakEvents(freshTrakAgencyID);
```

Comment:
I may be wrong but I think the TS compiler can tell what type of object is returned here implicitly

Reply:
You are correct.  I had seen examples in the code of explicitly showing the type in the caller but I guess that could be a hassle if the function changes what it returns.